src/adv.py

```diff
@@ -53,14 +53,96 @@
         "w_to": "",
     },
 """
+current_room = 'outside'
 
 # Write a class to hold player information, e.g. what room they are in currently
+class player_information:
+    def __init__(self, room):
+        self.name = room[current_room]['name']

```

Comment:
I feel like this should be self.room with an added parameter to handle self.name

Reply:
I might be misunderstanding how we're using the player class though. I don't particularly care about naming my player. :man_shrugging: